projects/www/src/app/components/docs/symbol-excerpt.component.ts

```diff
@@ -42,6 +42,9 @@ import { CodeHighlightPipe } from './code-highlight.pipe';
         font-family: 'Space Mono', monospace;
         font-variant-ligatures: none;
         font-size: 12px;
+        @media only screen and (max-width: 1280px) {
+          white-space: pre-wrap;
+        }
```

Comment:
For code parts, we can leave horizontal scrollbars. With pre-wrap, code won't be formatted and scroll bars will remain for empty content:

![1](https://github.com/user-attachments/assets/00755ea8-a5c9-4b30-9a2e-11580eb2af2b)